definitions/npm/@babel/code-frame_v7.x.x/flow_v0.30.x-/code-frame_v7.x.x.js

```diff
@@ -0,0 +1,66 @@
+declare module '@babel/code-frame' {
+  declare export type SourceLocation = {
+    start: {|
+      line: number,
+      column?: number,
+    |},
+    end?: {|
+      line: number,
+      column?: number,
+    |},
+    ...
```

Comment:
I guess we should scope this to an higher flow version, `flow_v0.84.x-` if I'm not mistaken.

Reply:
Done